apps/right-to-rent-check/index.js

```diff
@@ -46,7 +46,9 @@ module.exports = {
         }
       }]
     },
-    '/check-not-needed-location': {},
+    '/check-not-needed-location': {
+
```

Comment:
Did you mean this new line?

Reply:
yes I did. I just find it easier to read

Reply:
hmm, do we need the extra blank line could we not do instead?
```
check-not-needed-location: {
}
```